ruby/lib/minitest/queue/runner.rb

```diff
@@ -130,7 +131,7 @@ def report_command
       def run_tests_in_fork(queue)
         child_pid = fork do
           Minitest.queue = queue
-          Minitest::Reporters.use!([Minitest::Reporters::SpecReporter.new])
+          Minitest::Reporters.use!([Minitest::Reporters::BisectReporter.new])
```

Comment:
The minitest change basically broke `Minitest::Reporters::SpecReporter` so all the bisect integration test has a totally different output. SO while I was at it I implemented a simplified custom reporter for bisect.